README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
```

Comment:
+100 for "The mission of the Node.js Foundation is to enable and encourage the adoption of Node.js"

Instead of:
> the TSC is the home of the community driven portion of executing that mission

how about:
> the TSC is the body responsible for executing that mission on behalf of the community